master_data/models/base.py

```diff
@@ -0,0 +1,7 @@
+from django.db import models
+
+
+class BaseModelManager(models.Manager):
+
+    def get_queryset(self):
```

Comment:
IMO, we should not do this. In the future, maybe we want to show all data (including data that has deleted_flag=True). Let's just keep this in the business logic.

Reply:
Would you please tell me what the potential future case will be? As I'm adhering to the [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) principle and assuming that any data restoration should be done by an admin.

Reply:
From my perspective, we would like to have an admin page from the Front-end application side. As a result, the Vendor API should be able to return the Vendors that are deleted (`deleted_flag=True`) so that the front-end application side can still access it.

Applying this filter to the `BaseModelManager` limit our flexibility to control which queryset should be applied based on a particular role (e.g., whether it is for an admin user, vendor owner, or couple).